src/main.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(non_ascii_idents, pattern, plugin, test, question_mark)]
+#![feature(iter_arith, non_ascii_idents, pattern, plugin, test, question_mark)]
```

Comment:
`iter_arith` is [stable as of](https://github.com/rust-lang/rust/pull/34530) the July 3 nightly (which is part of 1.11.0, as [the awesome compiler warning will tell you](https://github.com/rust-lang/rust/commit/06c9e0f5f80d2a)).